helfi_api_base.module

```diff
@@ -50,3 +50,16 @@ function helfi_api_base_theme_suggestions_debug_item(array $variables) : array {
   $suggestions[] = 'debug_item__' . strtr($variables['id'], '.', '_');
   return $suggestions;
 }
+
+/**
+ * Implements hook_mail_alter().
+ */
+function helfi_api_base_mail_alter(&$message) : void {
+  $environmentResolver = \Drupal::service('helfi_api_base.environment_resolver');
+  try {
```

Comment:
This logic should maybe be documented in a comment. Something like: "Disables email sending on Hel.fi projects. Only Hel.fi instances can resolve an active environment, others throw a caught exception which skips this part."